In [1]:
import pandas as pd 

In [18]:
# Lendo cada aba
df_pedidos = pd.read_excel('Superloja.xls', sheet_name='Pedidos')
df_devolucoes = pd.read_excel('Superloja.xls', sheet_name='Devoluções')
df_pessoas = pd.read_excel('Superloja.xls', sheet_name='Pessoas')

# Juntando pedidos com devoluções (ID do pedido)
df_merge = pd.merge(df_pedidos, df_devolucoes, on='ID do pedido', how='left')

# Juntando com pessoas (Região)
df_final = pd.merge(df_merge, df_pessoas, on='Região', how='left')

In [19]:
 #  olhando as 5 primeiras linhas do dataset
df_final.head()

,ID da fila,ID do pedido,Data do pedido,Data de envio,Modo de envio,ID do cliente,Nome do cliente,Segmento,Cidade,Estado,...,ID do produto,Categoria,Sub-categoria,Nome do produto,Vendas,Quantidade,Desconto,Lucro,Devolvido,Gerente regional
0,1,MX-2021-143658,2021-10-02,2021-10-06,Classe padrão,SR-20575,Sofia Rocha,Varejo,Cidade do México,Distrito Federal,...,MAT-ET-10001014,Material de escritório,Etiquetas,"HON Etiquetas para organização de pastas, Ajus...",130.8,3,0.0,45.6,NaN,Vitór Rodrigues
1,4,MX-2019-155047,2019-10-16,2019-10-21,Classe padrão,LC-16570,Larissa Cardoso,Varejo,Dos Quebradas,Risaralda,...,MAT-FI-10003284,Material de escritório,Fichários,"Cardinal Pasta argolada, Transparente",354.4,4,0.0,49.6,NaN,Camila Fernandes
2,5,MX-2019-155047,2019-10-16,2019-10-21,Classe padrão,LC-16570,Larissa Cardoso,Varejo,Dos Quebradas,Risaralda,...,MAT-AR-10002762,Material de escritório,Arte,"Sanford Tela, Tamanhos variados",716.0,2,0.0,114.4,NaN,Camila Fernandes
3,6,MX-2019-155047,2019-10-16,2019-10-21,Classe padrão,LC-16570,Larissa Cardoso,Varejo,Dos Quebradas,Risaralda,...,MAT-EN-10000610,Material de escritório,Envelopes,"GlobeWeis Com lacre, Vermelho",561.2,2,0.0,213.2,NaN,Camila Fernandes
4,7,MX-2020-134096,2020-09-28,2020-10-02,Classe padrão,DG-13000,Diogo Goncalves,Varejo,São Paulo,São Paulo,...,MAT-EN-10000610,Material de escritório,Envelopes,"GlobeWeis Com lacre, Vermelho",561.2,2,0.0,213.2,NaN,Camila Fernandes


In [30]:
df_final["Devolvido"] = df_final["Devolvido"].notna().astype(int)

In [37]:
# (Agora: 1 = houve devolução ,  0 = não houve devolução)
df_final["Devolvido"].value_counts(normalize=True)

Devolvido
0    0.746403
1    0.253597
Name: proportion, dtype: float64

In [38]:
df_final.head()

,ID da fila,ID do pedido,Data do pedido,Data de envio,Modo de envio,ID do cliente,Nome do cliente,Segmento,Cidade,Estado,...,ID do produto,Categoria,Sub-categoria,Nome do produto,Vendas,Quantidade,Desconto,Lucro,Devolvido,Gerente regional
0,1,MX-2021-143658,2021-10-02,2021-10-06,Classe padrão,SR-20575,Sofia Rocha,Varejo,Cidade do México,Distrito Federal,...,MAT-ET-10001014,Material de escritório,Etiquetas,"HON Etiquetas para organização de pastas, Ajus...",130.8,3,0.0,45.6,0,Vitór Rodrigues
1,4,MX-2019-155047,2019-10-16,2019-10-21,Classe padrão,LC-16570,Larissa Cardoso,Varejo,Dos Quebradas,Risaralda,...,MAT-FI-10003284,Material de escritório,Fichários,"Cardinal Pasta argolada, Transparente",354.4,4,0.0,49.6,0,Camila Fernandes
2,5,MX-2019-155047,2019-10-16,2019-10-21,Classe padrão,LC-16570,Larissa Cardoso,Varejo,Dos Quebradas,Risaralda,...,MAT-AR-10002762,Material de escritório,Arte,"Sanford Tela, Tamanhos variados",716.0,2,0.0,114.4,0,Camila Fernandes
3,6,MX-2019-155047,2019-10-16,2019-10-21,Classe padrão,LC-16570,Larissa Cardoso,Varejo,Dos Quebradas,Risaralda,...,MAT-EN-10000610,Material de escritório,Envelopes,"GlobeWeis Com lacre, Vermelho",561.2,2,0.0,213.2,0,Camila Fernandes
4,7,MX-2020-134096,2020-09-28,2020-10-02,Classe padrão,DG-13000,Diogo Goncalves,Varejo,São Paulo,São Paulo,...,MAT-EN-10000610,Material de escritório,Envelopes,"GlobeWeis Com lacre, Vermelho",561.2,2,0.0,213.2,0,Camila Fernandes


In [39]:
taxa = df_final["Devolvido"].mean()
print(f"Taxa de devolução: {taxa:.2%}")

Taxa de devolução: 25.36%


In [41]:
# Lucro médio com e sem devolução | (Agora: 1 = houve devolução ,  0 = não houve devolução)
df_final.groupby("Devolvido")[["Vendas","Lucro"]].mean()

,Vendas,Lucro
Devolvido,,
0,2109.520268,216.739882
1,1976.962971,203.390527


In [45]:
# Proporção de devoluções por região e gerente
resultado = (
    df_final.groupby(["Região", "Gerente regional"])["Devolvido"]
    .mean()
    .sort_values(ascending=False)
)

print(resultado)

Região   Gerente regional 
Norte    Vitór Rodrigues      0.452003
Sul      Camila Fernandes     0.201754
Central  Aline Pinto          0.145733
Caribe   Beatriz Goncalves    0.118747
Name: Devolvido, dtype: float64


In [42]:
# Região com maior devolução

df_final.groupby("Região")["Devolvido"].mean().sort_values(ascending=False)

Região
Norte      0.452003
Sul        0.201754
Central    0.145733
Caribe     0.118747
Name: Devolvido, dtype: float64

In [46]:
# Categoria com maior devolução

df_final.groupby("Categoria")["Devolvido"].mean()

Categoria
Material de escritório    0.254970
Móveis                    0.256271
Tecnologia                0.246510
Name: Devolvido, dtype: float64

# Analise detalhada por gerente regional 



In [79]:
vitor_resultado = (
    df_final[df_final["Gerente regional"].str.strip() == "Vitór Rodrigues"]
    .groupby(["Categoria", "Região"])
    .agg(
        Quantidade=("Devolvido", "count"),
        Taxa_devolucao=("Devolvido", "mean")
    )
    .reset_index()
)

# Produto mais devolvido
produto_top = (
    df_final[(df_final["Gerente regional"].str.strip() == "Vitór Rodrigues") & (df_final["Devolvido"] == 1)]
    .groupby(["Categoria", "Nome do produto"])["Devolvido"]
    .count()
    .reset_index()
)
produto_top = produto_top.loc[produto_top.groupby("Categoria")["Devolvido"].idxmax()]

# Junta e ordena pela quantidade (maior primeiro)
vitor_resultado = pd.merge(
    vitor_resultado,
    produto_top[["Categoria", "Nome do produto"]],
    on="Categoria",
    how="left"
).rename(columns={"Nome do produto": "Produto mais devolvido"})

vitor_resultado = vitor_resultado.sort_values("Quantidade", ascending=False).reset_index(drop=True)
vitor_resultado["Ranking"] = range(1, len(vitor_resultado) + 1)

print(vitor_resultado[["Ranking", "Categoria", "Região", "Quantidade", "Taxa_devolucao", "Produto mais devolvido"]])

   Ranking               Categoria Região  Quantidade  Taxa_devolucao  \
0        1  Material de escritório  Norte        2230        0.451570   
1        2                  Móveis  Norte         980        0.454082   
2        3              Tecnologia  Norte         759        0.450593   

                Produto mais devolvido  
0            Acme Tesouras, Retangular  
1  Novimex Protetor de piso, Ajustável  
2              Hewlett Tinta, Vermelho  


In [80]:
camila_resultado = (
    df_final[df_final["Gerente regional"].str.strip() == "Camila Fernandes"]
    .groupby(["Categoria", "Região"])
    .agg(
        Quantidade=("Devolvido", "count"),
        Taxa_devolucao=("Devolvido", "mean")
    )
    .reset_index()
)

# Produto mais devolvido
produto_top = (
    df_final[(df_final["Gerente regional"].str.strip() == "Camila Fernandes") & (df_final["Devolvido"] == 1)]
    .groupby(["Categoria", "Nome do produto"])["Devolvido"]
    .count()
    .reset_index()
)
produto_top = produto_top.loc[produto_top.groupby("Categoria")["Devolvido"].idxmax()]

# Junta e ordena pela quantidade (maior primeiro)
camila_resultado = pd.merge(
    camila_resultado,
    produto_top[["Categoria", "Nome do produto"]],
    on="Categoria",
    how="left"
).rename(columns={"Nome do produto": "Produto mais devolvido"})

camila_resultado = camila_resultado.sort_values("Quantidade", ascending=False).reset_index(drop=True)
camila_resultado["Ranking"] = range(1, len(camila_resultado) + 1)

print(camila_resultado[["Ranking", "Categoria", "Região", "Quantidade", "Taxa_devolucao", "Produto mais devolvido"]])

   Ranking               Categoria Região  Quantidade  Taxa_devolucao  \
0        1  Material de escritório    Sul        1999        0.200100   
1        2                  Móveis    Sul         825        0.206061   
2        3              Tecnologia    Sul         710        0.201408   

               Produto mais devolvido  
0      Cardinal Divisórias, Reciclado  
1      HON Protetor de piso, Vermelho  
2  Motorola Smart Phone, Tamanho real  


In [81]:
beatriz_resultado = (
    df_final[df_final["Gerente regional"].str.strip() == "Beatriz Goncalves"]
    .groupby(["Categoria", "Região"])
    .agg(
        Quantidade=("Devolvido", "count"),
        Taxa_devolucao=("Devolvido", "mean")
    )
    .reset_index()
)

# Produto mais devolvido
produto_top = (
    df_final[(df_final["Gerente regional"].str.strip() == "Beatriz Goncalves") & (df_final["Devolvido"] == 1)]
    .groupby(["Categoria", "Nome do produto"])["Devolvido"]
    .count()
    .reset_index()
)
produto_top = produto_top.loc[produto_top.groupby("Categoria")["Devolvido"].idxmax()]

# Junta e ordena pela quantidade (maior primeiro)
beatriz_resultado = pd.merge(
    beatriz_resultado,
    produto_top[["Categoria", "Nome do produto"]],
    on="Categoria",
    how="left"
).rename(columns={"Nome do produto": "Produto mais devolvido"})

beatriz_resultado = beatriz_resultado.sort_values("Quantidade", ascending=False).reset_index(drop=True)
beatriz_resultado["Ranking"] = range(1, len(beatriz_resultado) + 1)

print(beatriz_resultado[["Ranking", "Categoria", "Região", "Quantidade", "Taxa_devolucao", "Produto mais devolvido"]])

   Ranking               Categoria  Região  Quantidade  Taxa_devolucao  \
0        1  Material de escritório  Caribe        1042        0.121881   
1        2                  Móveis  Caribe         408        0.120098   
2        3              Tecnologia  Caribe         369        0.108401   

                            Produto mais devolvido  
0  Stockwell Percevejo niquelado, Embalagem com 12  
1                  Bush Estante tradicional, Metal  
2                      Canon Fax sem fio, Vermelho  


In [ ]:
aline_resultado = (
    df_final[df_final["Gerente regional"].str.strip() == "Aline Pinto"]
    .groupby(["Categoria", "Região"])
    .agg(
        Quantidade=("Devolvido", "count"),
        Taxa_devolucao=("Devolvido", "mean")
    )
    .reset_index()
)

# Produto mais devolvido
produto_top = (
    df_final[(df_final["Gerente regional"].str.strip() == "Aline Pinto") & (df_final["Devolvido"] == 1)]
    .groupby(["Categoria", "Nome do produto"])["Devolvido"]
    .count()
    .reset_index()
)
produto_top = produto_top.loc[produto_top.groupby("Categoria")["Devolvido"].idxmax()]

# Junta e ordena pela quantidade (maior primeiro)
aline_resultado = pd.merge(
    aline_resultado,
    produto_top[["Categoria", "Nome do produto"]],
    on="Categoria",
    how="left"
).rename(columns={"Nome do produto": "Produto mais devolvido"})

aline_resultado = aline_resultado.sort_values("Quantidade", ascending=False).reset_index(drop=True)
aline_resultado["Ranking"] = range(1, len(aline_resultado) + 1)

print(aline_resultado[["Ranking", "Categoria", "Região", "Quantidade", "Taxa_devolucao", "Produto mais devolvido"]])

Ranking  Categoria               Região   Quantidade  Taxa_devolucao  Produto mais devolvido
1        Material de escritório  Norte    2230        0.451570        Acme Tesouras, Retangular
2        Móveis                  Norte     980        0.454082        Novimex Protetor de piso, Ajustável
3        Tecnologia              Norte     759        0.450593        Hewlett Tinta, Vermelho

In [85]:
# tabela já filtrada para Vítor Rodrigues
vitor_resultado = pd.DataFrame({
    "Categoria": ["Material de escritório", "Móveis", "Tecnologia"],
    "Região": ["Norte", "Norte", "Norte"],
    "Quantidade": [2230, 980, 759],
    "Taxa_devolucao": [0.451570, 0.454082, 0.450593],
    "Produto mais devolvido": [
        "Acme Tesouras, Retangular",
        "Novimex Protetor de piso, Ajustável",
        "Hewlett Tinta, Vermelho"
    ]
})

# Defina parâmetros financeiros
preco_medio = {
    "Material de escritório": 100,   # exemplo: R$ 100 por pedido
    "Móveis": 500,                   # exemplo: R$ 500 por pedido
    "Tecnologia": 300                # exemplo: R$ 300 por pedido
}
percentual_desconto = 0.10  # 10% de desconto nas devoluções

# Calcula devolvidos e não devolvidos
vitor_resultado["Devolvidos"] = (vitor_resultado["Quantidade"] * vitor_resultado["Taxa_devolucao"]).round()
vitor_resultado["Nao_devolvidos"] = vitor_resultado["Quantidade"] - vitor_resultado["Devolvidos"]

# Calcula valores financeiros
vitor_resultado["Preco_medio"] = vitor_resultado["Categoria"].map(preco_medio)
vitor_resultado["Faturamento_bruto"] = vitor_resultado["Quantidade"] * vitor_resultado["Preco_medio"]
vitor_resultado["Valor_devolvido"] = vitor_resultado["Devolvidos"] * vitor_resultado["Preco_medio"]
vitor_resultado["Desconto"] = vitor_resultado["Valor_devolvido"] * percentual_desconto
vitor_resultado["Faturamento_liquido"] = vitor_resultado["Faturamento_bruto"] - vitor_resultado["Valor_devolvido"] - vitor_resultado["Desconto"]

print(vitor_resultado[[
    "Categoria", "Região", "Quantidade", "Devolvidos", "Nao_devolvidos",
    "Faturamento_bruto", "Valor_devolvido", "Desconto", "Faturamento_liquido"
]])

                Categoria Região  Quantidade  Devolvidos  Nao_devolvidos  \
0  Material de escritório  Norte        2230      1007.0          1223.0   
1                  Móveis  Norte         980       445.0           535.0   
2              Tecnologia  Norte         759       342.0           417.0   

   Faturamento_bruto  Valor_devolvido  Desconto  Faturamento_liquido  
0             223000         100700.0   10070.0             112230.0  
1             490000         222500.0   22250.0             245250.0  
2             227700         102600.0   10260.0             114840.0  


## Vítor Rodrigues

In [91]:
# Filtra apenas Vítor Rodrigues
vitor_df = df_final[df_final["Gerente regional"].str.strip() == "Vitór Rodrigues"]

# Agrupa por categoria
vitor_resultado = vitor_df.groupby("Categoria").agg(
    Quantidade=("Quantidade", "sum"),
    Vendas=("Vendas", "sum"),
    Desconto_total=("Desconto", "sum"),
    Lucro_total=("Lucro", "sum"),
    Devolvidos=("Devolvido", "sum")
).reset_index()

# Calcula não devolvidos
vitor_resultado["Nao_devolvidos"] = vitor_resultado["Quantidade"] - vitor_resultado["Devolvidos"]

# Impacto financeiro das devoluções
vitor_resultado["Valor_devolvido"] = (
    vitor_resultado["Vendas"] * (vitor_resultado["Devolvidos"] / vitor_resultado["Quantidade"])
).round()

# Faturamento líquido (já considerando devoluções e descontos)
vitor_resultado["Faturamento_liquido"] = (
    vitor_resultado["Vendas"] - vitor_resultado["Valor_devolvido"] - vitor_resultado["Desconto_total"]
)
pd.set_option("display.float_format", "{:,.2f}".format)

print(vitor_resultado[[
    "Categoria", "Quantidade", "Vendas", "Devolvidos", "Nao_devolvidos",
    "Desconto_total", "Lucro_total", "Valor_devolvido", "Faturamento_liquido"
]])

                Categoria  Quantidade       Vendas  Devolvidos  \
0  Material de escritório        8467 2,321,193.00        1007   
1                  Móveis        3593 3,135,047.24         445   
2              Tecnologia        2841 3,382,018.48         342   

   Nao_devolvidos  Desconto_total  Lucro_total  Valor_devolvido  \
0            7460            0.00   549,257.00       276,065.00   
1            3148          247.20   124,596.84       388,282.00   
2            2499            0.51   721,993.68       407,128.00   

   Faturamento_liquido  
0         2,045,128.00  
1         2,746,518.04  
2         2,974,889.97  


> ## **Análise Regional — Norte | Vítor Rodrigues**

## Material de Escritório:
Pedidos: 8.467 (maior volume da região)
Devoluções: +1.000 unidades
Faturamento Líquido: R$ 2,045 milhões

## Móveis:
Faturamento Bruto: **R$ 3,1 milhões**

Devoluções: **R$ 388 mil**

Faturamento Líquido: **R$ 2,746 milhões**

## Tecnologia:

Faturamento Bruto: **R$ 3,382 milhões**

Devoluções: **R$ 407 mil**

Faturamento Líquido: **R$ 2,975 milhões**(maior entre as categorias)

## # Camila Fernandes (Sul)


In [100]:
camila_df = df_final[df_final["Gerente regional"].str.strip() == "Camila Fernandes"]

camila_resultado = camila_df.groupby("Categoria").agg(
    Quantidade=("Quantidade", "sum"),
    Vendas=("Vendas", "sum"),
    Desconto_total=("Desconto", "sum"),
    Lucro_total=("Lucro", "sum"),
    Devolvidos=("Devolvido", "sum")
).reset_index()

camila_resultado["Nao_devolvidos"] = camila_resultado["Quantidade"] - camila_resultado["Devolvidos"]
camila_resultado["Valor_devolvido"] = (
    camila_resultado["Vendas"] * (camila_resultado["Devolvidos"] / camila_resultado["Quantidade"])
).round()
camila_resultado["Faturamento_liquido"] = (
    camila_resultado["Vendas"] - camila_resultado["Valor_devolvido"] - camila_resultado["Desconto_total"]
)

print(camila_resultado[[
    "Categoria", "Quantidade", "Vendas", "Devolvidos", "Nao_devolvidos",
    "Desconto_total", "Lucro_total", "Valor_devolvido", "Faturamento_liquido"
]])

                Categoria  Quantidade       Vendas  Devolvidos  \
0  Material de escritório        7690 1,791,404.96         400   
1                  Móveis        3100 2,887,784.20         170   
2              Tecnologia        2603 2,458,264.87         143   

   Nao_devolvidos  Desconto_total  Lucro_total  Valor_devolvido  \
0            7290          380.40   103,604.76        93,181.00   
1            2930          159.50    94,560.60       158,362.00   
2            2460          124.03   164,617.07       135,049.00   

   Faturamento_liquido  
0         1,697,843.56  
1         2,729,262.70  
2         2,323,091.83  


###  Faturamento e Margem de Lucro

Categoria | Faturamento Líquido | Lucro | Margem 

## Material de escritório:

**R$ 1.697.843**

**R$ 103.604**

**6,1%** 

## Móveis:

**R$ 2.729.262**

**R$ 94.560** 

**3,5%** 

## Tecnologia: 

**R$ 2.323.091**

**R$ 164.617**

**7,1%** 


### Taxa de Devolução

Categoria | Devolvidos | Taxa | Valor Devolvido

## Material de escritório: 

**400** 

**5,2%** 

**R$ 93.181**

## Móveis:

**170**

**5,5%**

**R$ 158.362**

## Tecnologia: 

**143**

**5,5%**

**R$ 135.049**


###  Insights 

>1. **Material de escritório em colapso de margem** — caiu de 16,9% → 9,7% → 6,1% em três períodos consecutivos. Volume crescendo (3.850 → 7.045 → 7.690), mas lucro despencando. O crescimento está sendo feito a custo de rentabilidade.

>2. **Móveis com devolução crítica** — R$ 158.362 devolvidos é o maior valor absoluto entre todas as categorias e períodos. A margem voltou a cair para 3,5%, sinalizando que os problemas anteriores não foram resolvidos.

>3. **Tecnologia é a única categoria saudável** — lidera em lucro absoluto (R$ 164k) e tem a melhor margem (7,1%). Mesmo com queda gradual, é a mais consistente.

>4. ** Devoluções explodiram em todas as categorias** — as taxas subiram de ~3% para ~5,5% em todos os segmentos. Isso sugere um problema sistêmico, não isolado.

>5. **Descontos crescendo sem retorno** — Material de escritório: **R$ 119** → **R$ 288** → **R$ 380** em descontos, mas margem só cai. A política de desconto não está convertendo em lucro.

>**Recomendação urgente:** Revisar política de devoluções (problema sistêmico) e reavaliar estratégia de crescimento via desconto em Material de escritório — o volume está aumentando, mas destruindo valor.


##  Beatriz Goncalves (Caribe)


In [101]:

beatriz_df = df_final[df_final["Gerente regional"].str.strip() == "Beatriz Goncalves"]

beatriz_resultado = beatriz_df.groupby("Categoria").agg(
    Quantidade=("Quantidade", "sum"),
    Vendas=("Vendas", "sum"),
    Desconto_total=("Desconto", "sum"),
    Lucro_total=("Lucro", "sum"),
    Devolvidos=("Devolvido", "sum")
).reset_index()

beatriz_resultado["Nao_devolvidos"] = beatriz_resultado["Quantidade"] - beatriz_resultado["Devolvidos"]
beatriz_resultado["Valor_devolvido"] = (
    beatriz_resultado["Vendas"] * (beatriz_resultado["Devolvidos"] / beatriz_resultado["Quantidade"])
).round()
beatriz_resultado["Faturamento_liquido"] = (
    beatriz_resultado["Vendas"] - beatriz_resultado["Valor_devolvido"] - beatriz_resultado["Desconto_total"]
)

print(beatriz_resultado[[
    "Categoria", "Quantidade", "Vendas", "Devolvidos", "Nao_devolvidos",
    "Desconto_total", "Lucro_total", "Valor_devolvido", "Faturamento_liquido"
]])




                Categoria  Quantidade       Vendas  Devolvidos  \
0  Material de escritório        3850   964,486.24         127   
1                  Móveis        1543 1,384,882.80          49   
2              Tecnologia        1336 1,347,267.11          40   

   Nao_devolvidos  Desconto_total  Lucro_total  Valor_devolvido  \
0            3723          119.40   157,431.84        31,816.00   
1            1494           79.90    60,405.60        43,979.00   
2            1296           45.46   145,168.11        40,337.00   

   Faturamento_liquido  
0           932,550.84  
1         1,340,823.90  
2         1,306,884.65  


## Análise por Categoria

###  Volume de Vendas
**Material de escritório** domina em quantidade (3.850 unidades), representando **65%** do total de itens vendidos.
    
Móveis e Tecnologia têm volumes bem menores (1.543 e 1.336), mas com ticket médio muito mais alto.

###  Faturamento e Lucro

| Categoria | Faturamento Líquido | Lucro | Margem de Lucro |

Material de escritório: 

**R$: 932.550** 

**R$ 157.431** 

**16,9%** 

Móveis:

**R$ 1.340.823**

**R$:60.405**

**4,5%** 

Tecnologia:

**R$ 1.306.884**

**R$:145.168**

**11,1%** 

>  **Alerta: Móveis tem a maior receita, mas a pior margem de lucro(4,5%).**


###  Taxa de Devolução

Material de escritório: **3,3%** 
 
Móveis: **3,2%** 

Tecnologia:**3,0%**

>  As taxas são equilibradas entre as categorias, sem grandes discrepâncias.


###  Principais Insights

>  1. **Material de escritório é o carro-chefe de lucro** — apesar de menor receita que Móveis, gera 2,6x mais lucro. O volume alto e margens melhores fazem a diferença.

> 2. **Móveis merece atenção urgente** — é a categoria com maior receita **R$$: 1,34M**, mas entrega apenas **R$: 60.000** de lucro. Possíveis causas: descontos excessivos, custos altos ou alto valor de devoluções **R$: 43.979** — o maior entre as categorias).

> 3. **Tecnologia tem boa relação receita/lucro** — margem de 11,1% é saudável para o setor, com menor taxa de devolução.

> 4. **Descontos de Móveis são proporcionalmente altos** —  *R$$: 79,90 de desconto total
para 1.543 vendas vs **R$ 119,40** para 3.850 vendas em Material de escritório.


> **Recomendação prioritária:** Investigar a estrutura de custos e política de descontos em Móveis para elevar a margem de 4,5% para um patamar mais saudável.

# Aline Pinto (Central)


In [103]:

aline_df = df_final[df_final["Gerente regional"].str.strip() == "Aline Pinto"]

aline_resultado = aline_df.groupby("Categoria").agg(
    Quantidade=("Quantidade", "sum"),
    Vendas=("Vendas", "sum"),
    Desconto_total=("Desconto", "sum"),
    Lucro_total=("Lucro", "sum"),
    Devolvidos=("Devolvido", "sum")
).reset_index()

aline_resultado["Nao_devolvidos"] = aline_resultado["Quantidade"] - aline_resultado["Devolvidos"]
aline_resultado["Valor_devolvido"] = (
    aline_resultado["Vendas"] * (aline_resultado["Devolvidos"] / aline_resultado["Quantidade"])
).round()
aline_resultado["Faturamento_liquido"] = (
    aline_resultado["Vendas"] - aline_resultado["Valor_devolvido"] - aline_resultado["Desconto_total"]
)

print(aline_resultado[[
    "Categoria", "Quantidade", "Vendas", "Devolvidos", "Nao_devolvidos",
    "Desconto_total", "Lucro_total", "Valor_devolvido", "Faturamento_liquido"
]])

                Categoria  Quantidade       Vendas  Devolvidos  \
0  Material de escritório        7045 1,771,303.04         300   
1                  Móveis        2714 2,439,855.58          92   
2              Tecnologia        2472 2,376,680.06          93   

   Nao_devolvidos  Desconto_total  Lucro_total  Valor_devolvido  \
0            6745          288.40   165,046.24        75,428.00   
1            2622          111.20   209,000.18        82,707.00   
2            2379          102.75   203,252.86        89,414.00   

   Faturamento_liquido  
0         1,695,586.64  
1         2,357,037.38  
2         2,287,163.31  


###  Faturamento e Margem de Lucro

Categoria | Faturamento Líquido | Lucro | Margem

Material de escritório: 

**R$ 1.695.586**

**R$ 165.046**

**9,7%**

Móveis:
**R$ 2.357.037**

**R$ 209.000** 

**8,9%** 

Tecnologia:

**R$ 2.287.163**

**R$ 203.252**

**8,9%** 



###  Taxa de Devolução

Categoria | Devolvidos | Taxa 
Material de escritório:
**300**
**4,3%**

Móveis: 
**92**
**3,4%**

Tecnologia:
**93** 
**3,8%**



### Insights

>1. **Móveis virou destaque positivo** — a margem saltou de 4,5% para 8,9%, sinalizando melhora expressiva na rentabilidade. A categoria agora lidera em lucro absoluto (R$ 209k).

>2. **Material de escritório perdeu eficiência** — margem caiu de 16,9% para 9,7%. Com volume alto e muitas devoluções (300 itens, a maior taxa: 4,3%), o custo operacional está pesando.

>3. **Tecnologia segue estável** — margem consistente (~11% → 8,9%), com o maior valor devolvido em reais (R$ 89.414), o que merece monitoramento.

>4. **Devoluções em valor absoluto cresceram em todas as categorias** — Tecnologia lidera com R$ 89k devolvido. Revisão de política de troca/devolução pode impactar diretamente o lucro.

>5. **As três categorias convergem para margens parecidas (~9%)** — o negócio está mais equilibrado, mas também menos diferenciado.


> **Ponto crítico:** O aumento em devolucões de Material de escritório (127 → 300) é o maior sinal de alerta. Investigar as causas pode recuperar margem perdida.